base/complex.jl

```diff
@@ -75,17 +75,66 @@ real(T::Type) = typeof(real(zero(T)))
 real{T<:Real}(::Type{T}) = T
 real{T<:Real}(::Type{Complex{T}}) = T
 
+"""
+    isreal(x) -> Bool
+
+Test whether `x` or all its elements are numerically equal to some real number.
+
+```jldoctest
+julia> isreal(5.)
+true
+
+julia> isreal([4.; complex(0,1)])
+false
+```
+"""
 isreal(x::Real) = true
 isreal(z::Complex) = iszero(imag(z))
+
 """
     isimag(z) -> Bool
 
 Test whether `z` is purely imaginary, i.e. has a real part equal to 0.
 """
 isimag(z::Number) = iszero(real(z))
+
+"""
+    isinteger(x) -> Bool
+
+Test whether `x` or all its elements are numerically equal to some integer.
+
+```jldoctest
+julia> isinteger(4.0)
+true
+
+julia> isinteger([1; 2; 5.5])
+false
+```
+"""
```

Comment:
This docstring is more general than the signature `z::Complex` so should probably be moved to a more suitable method definition if one exists.

(Same thing applies to a couple of others below this.)